sdcm/nemesis.py

```diff
@@ -2965,14 +2965,40 @@ def _write_read_data_to_multi_dc_keyspace(self, datacenters: List[str]) -> None:
         read_thread = self.tester.run_stress_thread(stress_cmd=read_cmd, use_single_loader=True)
         self.tester.verify_stress_thread(cs_thread_pool=read_thread)
 
+    def _switch_to_network_replication_strategy(self, keyspaces: List[str]) -> None:
+        """Switches replication strategy to NetworkTopology for given keyspaces. Sets rf to all nodes.
+        Switching replication strategy from Simple to Network requires stopping traffic and repair """
+        status = self.tester.db_cluster.get_nodetool_status()
+        assert len(status) == len(self.tester.db_cluster.params.get("n_db_nodes").split(" ")), \
+            f"status returned wrong number or dc's: {status}"
+        full_replication = NetworkTopologyReplicationStrategy(**{dc: len(status[dc].keys()) for dc in status})
+        # stop traffic
+        for node in self.cluster.nodes:
+            node.run_nodetool(sub_cmd="drain", publish_event=True)
+        # repair cluster
+        for node in self.cluster.nodes:
+            node.run_nodetool(sub_cmd="repair", publish_event=True)
+        # switch replication strategy
+        for keyspace in keyspaces:
+            full_replication.apply(self.target_node, keyspace)
+        # again repair after switch
+        for node in self.cluster.nodes:
+            node.run_nodetool(sub_cmd="repair", publish_event=True)
+        # cleanup
+        for node in self.cluster.nodes:
+            node.run_nodetool(sub_cmd="cleanup", publish_event=True)
```

Comment:
This part of the nemesis may make it very long nemesis in some of the longevities we have.
Please test it on a longevity like the 1TB.

Reply:
actually this procedure is wrong. I've contacted dev for the proper one (cannot do repair when node is 'drain'ed). Working on it.

Reply:
I think there is a flag for allowing dead nodes.